### ISSUE TITLE
`test_benchdnn_conv_ci_cpu` and `test_benchdnn_reorder_ci_cpu` failing on AArch64

### ISSUE THREAD

**Issue by milpuz01** [oneapi-src/oneDNN#1167]
# Summary
The [recent change](https://github.com/oneapi-src/oneDNN/commit/6e9680e5586d3a0724b8e8969a53273788892cf4) makes `test_benchdnn_conv_ci_cpu` and `test_benchdnn_reorder_ci_cpu' fail. [This change](https://github.com/oneapi-src/oneDNN/commit/6b915e43e53977008a1545056a9b388456b51a6f) reverts when dimension is 2, but it would need to be reverted for other dimensions too.

# Version
Current head of master: [4b97249](https://github.com/oneapi-src/oneDNN/commit/4b9724917e17e75b5e9aca1fb9e7e9e5fbad5d26)

# Environment
* Arm Neoverse-N1 (AArch64, Arm-v8.2a)
* GCC9 and Clang9
* Ubuntu 18.04

# Steps to reproduce
```
mkdir -p build
cd build
cmake -DDNNL_CPU_RUNTIME=OMP -DDNNL_BUILD_FOR_CI=ON ../. && make -j
ctest -R test_benchdnn_conv_ci_cpu --verbose
ctest -R test_benchdnn_reorder_ci_cpu --verbose
```

# Observed behavior
```
99: run: --conv --dir=FWD_I --cfg=u8s8s32 --attr-zero-points=dst:common:1+src:common:2 g1ic3iw5oc16ow5kw3pw1n"1d_conv:1st"
99: error [int execute_reorder(const dnn_mem_t &, dnn_mem_t &, const_dnnl_primitive_attr_t):88]: 'dnnl_reorder_primitive_desc_create(&r_pd_, &src.md_, src.engine(), &dst.md_, dst.engine(), attr)' -> unimplemented(3)
1/1 Test #99: test_benchdnn_conv_ci_cpu ........***Failed  110.35 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) = 110.39 sec

The following tests FAILED:
         99 - test_benchdnn_conv_ci_cpu (Failed)
Errors while running CTest
```

```
109: 18144:UNIMPLEMENTED __REPRO: --reorder --sdt=f32 --ddt=s8 --stag=abx --dtag=xba --oflag=s8s8_comp:1 2x16x3x4
109: run: --reorder --sdt=s8 --ddt=s8 --stag=abx --dtag=xba --oflag=s8s8_comp:1 2x16x3x4
109: 18145:UNIMPLEMENTED __REPRO: --reorder --sdt=s8 --ddt=s8 --stag=abx --dtag=xba --oflag=s8s8_comp:1 2x16x3x4
109: run: --reorder --sdt=bf16 --ddt=s8 --stag=abx --dtag=xba --oflag=s8s8_comp:1 2x16x3x4
109: 18146:SKIPPED (DATA_TYPE_NOT_SUPPORTED) __REPRO: --reorder --sdt=bf16 --ddt=s8 --stag=abx --dtag=xba --oflag=s8s8_comp:1 2x16x3x4
109: run: --reorder --sdt=f32 --ddt=s8 --stag=abx --dtag=xba --oflag=s8s8_comp:1 1x17x5x3
109: 18147:UNIMPLEMENTED __REPRO: --reorder --sdt=f32 --ddt=s8 --stag=abx --dtag=xba --oflag=s8s8_comp:1 1x17x5x3
109: run: --reorder --sdt=s8 --ddt=s8 --stag=abx --dtag=xba --oflag=s8s8_comp:1 1x17x5x3
109: 18148:UNIMPLEMENTED __REPRO: --reorder --sdt=s8 --ddt=s8 --stag=abx --dtag=xba --oflag=s8s8_comp:1 1x17x5x3
109: run: --reorder --sdt=bf16 --ddt=s8 --stag=abx --dtag=xba --oflag=s8s8_comp:1 1x17x5x3
109: 18149:SKIPPED (DATA_TYPE_NOT_SUPPORTED) __REPRO: --reorder --sdt=bf16 --ddt=s8 --stag=abx --dtag=xba --oflag=s8s8_comp:1 1x17x5x3
109: tests:18150 passed:5376 skipped:12770 mistrusted:0 unimplemented:4 failed:4 listed:0
109: total compute_ref: sum(s):0.00
1/1 Test #109: test_benchdnn_reorder_ci_cpu .....***Failed    9.53 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =   9.59 sec

The following tests FAILED:
        109 - test_benchdnn_reorder_ci_cpu (Failed)
Errors while running CTest
```

# Expected behavior

All tests passing

**Comment by igorsafo** [comment]
Hi @milpuz01 ,
Thanks for your issue. I was able to reproduce it. We will notify you once the issue is solved.

**Comment by nSircombe** [comment]
Hi @igorsafo,
I was just wondering if there had been any progress on a resolution for this bug?

**Comment by vpirogov** [comment]
Hi @nSircombe, 

the fix is in master and rls-v2.5 branches.

**Comment by nSircombe** [comment]
Thanks @vpirogov - they're passing for us on AArch64 again.
